src/routers/index.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const router = express.Router();
+
+const bookRouter = require("./bookRouter");
+router.use("/books", bookRouter);
```

Comment:
마이너하지만 가독성을 위해 한줄 띄어주면 좋겠네요!
```suggestion
const bookRouter = require("./bookRouter");

router.use("/books", bookRouter);
```